README.md

```diff
@@ -12,7 +12,7 @@ For Play 2.6.x use version [![Maven Central](https://maven-badges.herokuapp.com/
 ```
 libraryDependencies += "com.gu" %% "play-googleauth" % "0.7.0"
 ```
-This version has a dependancy on `cats 0.9.*`. Therefore, (for compatibility) it is advised your project or any of its dependendancies do not depend on `cats 0.7.*` or below.
+This version has a dependancy on `cats 0.9.*`. Therefore, (for compatibility) it is advised your project or any of its dependendencies do not depend on `cats 0.7.*` or below.
```

Comment:
you didn't fix the thing I was laughing at!
`dependendencies` should be
`dependencies`